os/contiki-default-conf.h

```diff
@@ -107,8 +107,8 @@
 #define NBR_TABLE_CONF_CAN_ACCEPT_NEW          rpl_nbr_can_accept_new
 #endif /* NBR_TABLE_CONF_CAN_ACCEPT_NEW */
 /* Leave 3 spots for candidate parents incl. default route. */
-#if RPL_NBR_POLICY_MAX_NEXTHOP_NEIGHBORS
-#define RPL_NBR_POLICY_MAX_NEXTHOP_NEIGHBORS  MAX(NBR_TABLE_CONF_MAX_NEIGHBORS - 3, 0)
+#ifndef RPL_NBR_POLICY_CONF_MAX_NEXTHOP_NEIGHBORS
+#define RPL_NBR_POLICY_CONF_MAX_NEXTHOP_NEIGHBORS  MAX(NBR_TABLE_CONF_MAX_NEIGHBORS - 3, 0)
 #endif /* RPL_NBR_POLICY_MAX_NEXTHOP_NEIGHBORS */
```

Comment:
Comment needs to be updated to match the new name of the variable.

Reply:
Fixed